java/src/jmri/jmrit/vsdecoder/Steam1Sound.java

```diff
@@ -289,6 +294,15 @@ public void setXml(Element e, VSDFile vf) {
         }
         log.debug("engine gain: {}", engine_gain);
 
+        // Allows to handle dynamic gain for chuff sounds
+        n = e.getChildText("dynamic-gain"); // Optional value
+        if ((n != null) && (n.equals("yes"))) {
+            is_dynamic_gain = true;
+        } else {
+            is_dynamic_gain = false; // Muss das nicht oben stehen (s.a. waiting-active)?
```

Comment:
It would be better to have Java comments in English than German.

Reply:
This can be fixed in another PR. I will merge this PR once 24 hours has passed.

Reply:
All right. I will correct it with my next PR. Thank you!